### ISSUE TITLE
fix build: Rename files

### ISSUE THREAD

**Issue by abranhe** [AllAlgorithms/java#71]
https://travis-ci.org/AllAlgorithms/java/builds/565297462

<img width="504" alt="Screen Shot 2019-07-30 at 12 24 35 AM" src="https://user-images.githubusercontent.com/21347264/62100635-79b4b000-b260-11e9-92b7-6ef9ff7c9571.png">

**Comment by jfullstackdev** [comment]
yup, this one should be addressed too, just like `heapSort.java` and `HeapSort.java`
